frontend/src/scenes/heatmaps/heatmapsBrowserLogic.ts

```diff
@@ -50,6 +50,7 @@ export const heatmapsBrowserLogic = kea<heatmapsBrowserLogicType>([
         // TRICKY: duplication ends
         setIframeWidth: (width: number | null) => ({ width }),
         toggleFilterPanelCollapsed: true,
+        setIframeError: (error: string | null) => ({ error }),
```

Comment:
this is a string since i'm going to follow-up with a change that adds an error if we time-out trying to load the iframe